shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/survey/SurveyService.kt

```diff
@@ -29,4 +31,13 @@ internal class SurveyServiceImpl: SurveyService, KoinComponent {
             }
         )
     }
+
+    override fun getSurvey(id: String): Flow<SurveyDetailApiModel> {
+        return apiService.performRequest(
+            HttpRequestBuilder().apply {
```

Comment:
`HttpRequestBuilder().apply` is repeated. I think we can bring the HttpRequestBuilder setup to the ApiClient.kt. We only need to pass the URL string, body, and method.

Reply:
Is it optional? it will require some refactoring and updating on other services.
I will try to improve it when I have more time, but currently, the deadline is pretty tight, so I suggest we focus more on essential parts 🙏